apps/bfd-db-migrator/src/test/java/gov/cms/bfd/migrator/app/MigratorAppIT.java

```diff
@@ -242,21 +290,29 @@ public void testMigrationRunWhenSchemaFileErrorExpectFailedExitCodeAndMigrations
   }
 
   /**
-   * Gets the number of migration scripts by checking the directory they are located in.
+   * Gets the number of migration scripts by checking the directory they are located in and counting
+   * the files.
    *
    * @return the number migration scripts
    */
-  public int getNumMigrationScripts() {
-    String migrationPath = "src/main/resources/db/migration";
-
-    long fileCount = 0;
-    try (Stream<Path> files = Files.list(Paths.get(migrationPath))) {
-      fileCount = files.count();
-    } catch (IOException io) {
-      fail("Could not find file path for migration scripts at: " + migrationPath);
+  public int getNumMigrationScripts() throws IOException {
+
+    JarFile migrationJar =
+        new JarFile(
+            new File(
+                "target/db-migrator/bfd-db-migrator-1.0.0-SNAPSHOT/lib/bfd-model-rif-1.0.0-SNAPSHOT.jar"));
```

Comment:
this is going to cause an issue once we have versioning which is slated for the surge. there is some other code where we start up the pipeline and server that gets around this. its ugly but it works.

Reply:
Fixed to use a search that doesn't need the snapshot information, which should work with any future versioning.